### ISSUE TITLE
Build package for win64

### ISSUE THREAD

**Issue by Ayuus** [FactoryXCode/MfPack#29]
In order to build your package for use in 64-bits applications, could you please add Win64 as platform, and apply changes in attached version of WinApi.DirectX.DxVa9Typ.pas (RECT ->TRect)
[WinApi.DirectX.DXVA9Typ.zip](https://github.com/FactoryXCode/MfPack/files/8482753/WinApi.DirectX.DXVA9Typ.zip)
?

**Comment by FactoryXCode** [comment]
Thank you.

You should add the Win64 platform within the IDE. 
By default this is Win32.  (Just Right-click 'Target Platforms (Win32)' to add the Win64 platform.)

The Delphi Package file (.dpk) does not store platform information. This is stored in the Delphi Project File (.dproj)

Regards, Tony.

**Comment by Ayuus** [comment]
Hi Tony, 

No, Win64 can also be added as platform to packages. That has the advantage of uncoupling package building from application building, for both Win32 and Win64. Applications do not need access to package source code.

And indeed that should be done in the .dproj of the package. 
Refer to attached package project file (in which I added a dcu output directory per configuration/platform).  
However, of course it was just a suggestion, no more than that. 

Best regards, Ronald

**Comment by FactoryXCode** [comment]
Hi Ronald,

Thanks for your reply concerning this matter.

In the past our team had this discussion, if this should be implemented.
The conclusion was that compatibility with older Delphi compilers would mean we had to write code for every compiler version and spends lots of time to debug and/or create a compatible version.
So, we made a choice to write code as flexible as possible  users can adjust the compiler and project settings where they are comfortable with. That's also the reason we implemented the .inc files for fine tuning and alignment settings per unit for 32 or 64 bit. 

Remember the API is translated from the Win32 API ( a Win64 API does not exist ;-) ), Windows internally is taking care about 64 or 32 bit behavior by default if nothing is defined. There is no reason to compile this API for the 64 bit platform.

Regards, Tony.
